flink-contrib/flink-storm-compatibility/flink-storm-compatibility-core/src/main/java/org/apache/flink/stormcompatibility/wrappers/AbstractStormSpoutWrapper.java

```diff
@@ -97,9 +103,21 @@ public AbstractStormSpoutWrapper(final IRichSpout spout,
 	}
 
 	@Override
+	public void open(Configuration parameters) throws Exception {
+		/* parameters is task configuration, we can get storm configuration only from job configuration */
+		RuntimeContext ctx = super.getRuntimeContext();
+		if (ctx instanceof StreamingRuntimeContext)
+		{
+			Configuration jobConfiguration = ((StreamingRuntimeContext) ctx).getJobConfiguration();
+			stormConf = new HashMap<String, Object>();
+			stormConf.putAll(jobConfiguration.getConfData());
```

Comment:
I will add null test.In fact, jobConfiguration can not be null. Because StromBoltWrapper unit test mock StreamingRuntimeContext object, so its jobConfiguration  can be null.